dev/com.ibm.ws.http.plugin.merge/src/com/ibm/ws/http/plugin/merge/internal/PluginMergeToolImpl.java

```diff
@@ -188,8 +190,8 @@ private void printMergedCopy(String output) throws IOException, ParserConfigurat
         DocumentBuilderFactory dbf = DocumentBuilderFactory.newInstance();
         DocumentBuilder parser = dbf.newDocumentBuilder();
         Document mergeDoc = parser.newDocument();
-        final Comment comment = mergeDoc.createComment(" This config file was generated by pluginUtility merge v1.0.73 on " +
-                                                       dateTimeFormatter.format(ZonedDateTime.now()) + " ");
+        final Comment comment = mergeDoc.createComment(" This config file was generated by pluginUtility merge v1.0.75 on " +
+                                                       new SimpleDateFormat("yyyy.MM.dd 'at' HH:mm:ss z").format(new Date()) + " ");
```

Comment:
Why did we revert this? https://github.com/OpenLiberty/open-liberty/commit/a0d92abfc51c367a89d0e680d7c2ceac697c4ee2